pallets/pips/src/lib.rs

```diff
@@ -1573,6 +1565,34 @@ impl<T: Trait> Module<T> {
     }
 }
 
+mod weight_helper {
```

Comment:
We can directly declare this function as you did for others, instead of having an extra module.